docs/openapi/api-statistics-interop-v1.yaml

```
@@ -15,7 +15,7 @@ servers:
   - url: https://to-define/
     description: Development environment
 paths:
-  '/eservices/{eserviceRecordId}':
+  '/eservices/telemetryData/{eserviceRecordId}':
```

being all endpoints of this services part of the `telemetryData`, the correct path would be
```suggestion
  '/telemetryData/eservices/{eserviceRecordId}':
```